src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -637,7 +647,9 @@ private static void finish(@CheckForNull final String cookie) {
                         // Cf. AbstractBuild.getEnvironment:
                         env.put("WORKSPACE", workspace.getRemote());
                         FlowNode flowNode = context.get(FlowNode.class);
-                        flowNode.addAction(new WorkspaceActionImpl(workspace, flowNode));
+                        if (flowNode != null) {
```

Comment:
In practice, it's never null, but since `context.get` isn't `Nonnull`, my IDE is convinced it *could* be null, so I prefer to shut up the IDE and play it safe. =)